components/main_menu/index.tsx

```diff
@@ -54,6 +56,15 @@ function mapStateToProps(state: GlobalState) {
     const moreTeamsToJoin = joinableTeams && joinableTeams.length > 0;
     const rhsState = getRhsState(state);
 
+    const subscription = selectCloudSubscription(state);
+    const adminAnalytics = getAdminAnalytics(state);
+    const cloudLimits = selectCloudLimits(state);
+
+    const isCloudFreeEnabled = cloudFreeEnabled(state);
+    const isFreeTrial = subscription?.is_free_trial === 'true';
+
+    const teamsLimitReached = (adminAnalytics?.TOTAL_TEAMS ?? NaN) >= (cloudLimits.teams?.active ?? NaN);
```

Comment:
Here I believe we should use the limits usage like here: 

https://github.com/mattermost/mattermost-webapp/blob/3c094bb558bc589de1b27de6ae61952f447a7ddd/components/common/hooks/useGetUsageDeltas.ts#L31

The analitycs total teams is probably returning the active teams but just maybe we should still to the same get limits usage check to be consistent

https://github.com/mattermost/mattermost-webapp/blob/3c094bb558bc589de1b27de6ae61952f447a7ddd/components/common/hooks/useGetUsageDeltas.ts#L12

Reply:
@julmondragon do we need to update this with the team usage instead?